sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaReadSchemaTransformConfiguration.java

```diff
@@ -39,7 +39,7 @@ public abstract class KafkaReadSchemaTransformConfiguration {
 
   public static final Set<String> VALID_START_OFFSET_VALUES = Sets.newHashSet("earliest", "latest");
 
-  public static final String VALID_FORMATS_STR = "AVRO,JSON";
+  public static final String VALID_FORMATS_STR = "raw,avro,json";
   public static final Set<String> VALID_DATA_FORMATS =
       Sets.newHashSet(VALID_FORMATS_STR.split(","));
```

Comment:
Wanting to look more at this:

* Is there a reason for the convention of UPPERCASE, vs lower?  
* Why use a string that we need to then split on the subsequent line, instead of adding to a new set directly?

Reply:
unless that is more of a java convention to not have to constuct the Hash/Set.

Reply:
Context on the first point --> I'm used to FINALS being UPPERCASE

Reply:
Agreed. I tried to match: https://github.com/apache/beam/blob/7531501ff27f53bcb4fcd6942f34dbd45665805d/sdks/python/apache_beam/yaml/yaml_io.py#L195 so all the IOs follow the same standard. 
Regarding: `unless that is more of a java convention to not have to constuct the Hash/Set.` I'll do some research but I don't think there is any. I imagine the reason why it was made this way was to prevent misalignments between the print of `Valid data formats` and the Set of formats.

Reply:
+1 to ensuring the two remain consistent (though one could just as easily join for the error message).